Resources/translations/SonataDashboardBundle.en.xliff

```diff
@@ -214,6 +214,22 @@
                 <source>form.label_template</source>
                 <target>Template</target>
             </trans-unit>
+            <trans-unit id="53" resname="form.label_title">
```

Comment:
For the German file:

form.label_title: Titel
form.label_url: URL
form.label_icon: Icon
form.label_color: Farbe